docker-bench-security.sh

```diff
@@ -59,7 +59,7 @@ if [ -z "$logger" ]; then
 fi
 
 yell "# ------------------------------------------------------------------------------
-# Docker Bench for Security v1.0.0
+# Docker Bench for Security v1.1.0
```

Comment:
I guess that would make sense, but are we actually tracking Docker's version?
We're using the latest CIS document as a base, regardless of the Docker version, and we're making changes when a new CIS document is released.